viewing_party/party.py

```diff
@@ -0,0 +1,199 @@
+##Wave 1!!
+
+
+def create_movie(title,genre,rating):
+    
+    if bool(title)==True and bool(genre)==True and bool(rating)==True:
+        dict={"title":"Title A","genre":"Horror","rating":3.5}
+        return dict
+    else:
+        return None
+
+def add_to_watched(user_data,movie):
+   
+    #user_data={'watched':[{},{},{}]}
+    #movie={"title": "Title A", "genre": "Horror", "rating": 3.5}
+    watched=user_data['watched']
+    watched.append(movie)
```

Comment:
great implementation! you could also make it a one-liner user_data["watched"].append(movie)